rmgpy/molecule/molecule.py

```diff
@@ -669,6 +675,9 @@ class Molecule(Graph):
     `symmetryNumber`        ``int``     The (estimated) external + internal symmetry number of the molecule
     `multiplicity`          ``int``     The multiplicity of this species, multiplicity = 2*total_spin+1
     `props`                 ``dict``    A list of properties describing the state of the molecule.
+    `InChI`                 ``str``     A string representation of the molecule in InChI
+    `atoms`                 ``list``    A list of Atom objects in the molecule
+    `_fingerprint`          ``str``     A representation for fast comparison, set as molecular formula
```

Comment:
I don't think you should include `_fingerprint` since it's intended to be a private attribute. The `getFingerprint` method should be used to access it.